crypto/bls.go

```diff
@@ -243,18 +247,7 @@ const identityBLSSignatureHeader = byte(0xC0)
 // suspected to be equal to identity, which avoids failing the aggregated
 // signature verification.
 func IsBLSSignatureIdentity(s Signature) bool {
```

Comment:
maybe undo changes to this function.  the original version has no gc overhead.  the new one does

Reply:
Interesting! Is it because of the use of the global variable `identityBLSSignature` ? The global aspect of the variable doesn't make the gc overhead negligible?

Reply:
IMHO, the new version simpler, so it should be fine, since the GC overhead is constant.